### ISSUE TITLE
EndOfLine cop broken for UTF-8 files

### ISSUE THREAD

**Issue by marten** [rubocop/rubocop#668]
Example file (`app/controllers/epd/reports_controller.rb`): 

``` ruby
# encoding: UTF-8
class Epd::ReportsController < EpdAreaController
  'terecht bij uw ROM-coördinator.'
end
```

Output from Rubocop:

```
Inspecting 1 file
Scanning /Users/marten/rgoc/roqua/app/controllers/epd/reports_controller.rb
For /Users/marten/rgoc/roqua/app/controllers/epd: configuration from /Users/marten/rgoc/roqua/.rubocop.yml
Inheriting configuration from /Users/marten/rgoc/roqua/rubocop-todo.yml
Default configuration from /Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/config/default.yml
Inheriting configuration from /Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/config/enabled.yml
Inheriting configuration from /Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/config/disabled.yml
An error occurred while EndOfLine cop was inspecting /Users/marten/rgoc/roqua/app/controllers/epd/reports_controller.rb.
invalid byte sequence in US-ASCII
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cop/style/end_of_line.rb:13:in `block in investigate'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cop/style/end_of_line.rb:12:in `each'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cop/style/end_of_line.rb:12:in `each_with_index'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cop/style/end_of_line.rb:12:in `investigate'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cop/commissioner.rb:82:in `block (2 levels) in invoke_cops_callback'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cop/commissioner.rb:88:in `with_cop_error_handling'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cop/commissioner.rb:81:in `block in invoke_cops_callback'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cop/commissioner.rb:79:in `each'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cop/commissioner.rb:79:in `invoke_cops_callback'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cop/commissioner.rb:55:in `investigate'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cop/team.rb:42:in `inspect_file'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cli.rb:138:in `inspect_file'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cli.rb:77:in `block in process_files'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cli.rb:71:in `each'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cli.rb:71:in `process_files'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/lib/rubocop/cli.rb:32:in `run'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/bin/rubocop:14:in `block in <top (required)>'
/Users/marten/.rvm/rubies/ruby-2.0.0-p247/lib/ruby/2.0.0/benchmark.rb:296:in `realtime'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/gems/rubocop-0.15.0/bin/rubocop:13:in `<top (required)>'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/bin/rubocop:19:in `load'
/Users/marten/.rvm/gems/ruby-2.0.0-p247/bin/rubocop:19:in `<main>'
.

1 file inspected, no offences detected

1 error occurred:
An error occurred while EndOfLine cop was inspecting /Users/marten/rgoc/roqua/app/controllers/epd/reports_controller.rb.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.15.0 (using Parser 2.0.0, running on ruby 2.0.0 x86_64-darwin12.3.0)
Finished in 0.013773 seconds
```
